recipes/aws-sdk-cpp/run_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Stop on first error
+set -e
+
+if [ "$(uname)" == "Darwin" ]; then
+    # OSX
+    clang++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib" -Wl,-rpath,"${PREFIX}/lib" -laws-cpp-sdk-core -o test-core
+else
+    # Linux
+    g++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib${ARCH}" -Wl,-rpath,"${PREFIX}/lib${ARCH}" -laws-cpp-sdk-core -o test-core
```

Comment:
Add `-DCMAKE_INSTALL_LIBDIR=lib` when building and remove `${ARCH}`. We are using `${PREFIX}/lib` for all packages.

Reply:
ok, updated